File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.cpp
class Solution {
public:
    bool ispalindrome(string words){
        int i=0;
        int j=words.size()-1;
        while(i<=j){
            if(words[i]!=words[j]){
                return false;
            }
            i++;
            j--;
        }
        return true;
    }
    string firstPalindrome(vector<string>& words) {
        int n=words.size();
        for(int i=0;i<n;i++){
            if(ispalindrome(words[i])){
                return words[i];
                break;
            }
        }
        return "";
    }
};

File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.java
class Solution {
    public String firstPalindrome(String[] words) {
        int n=words.length;
        String ans="";
        for(int i=0;i<n;i++){
            StringBuilder sb = new StringBuilder(words[i]);
            sb.reverse();

            if(words[i].equals(sb.toString())){
                ans=words[i];
                break;
            }
        }
        return ans;
    }
}